app/models/federation_config.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class FederationConfig < ActiveRecord::Base
+  belongs_to :collection
+
+  delegate :provider_enum, to: :class
+
+  class << self
+    def provider_enum
+      Foederati::Providers.registry.keys
+    end
+  end
+
+  validates_presence_of %i{collection_id provider}
```

Comment:
To keep this consistent, can we use the `validates` format for the presence validator too?